.cirrus.yml

```diff
@@ -70,6 +70,7 @@ cross_task:
         image: catalina-xcode
     setup_script: |
         export PATH=$GOPATH/bin:$PATH
+        brew update
```

Comment:
Oh, okay, this is new to me.  I'm fine if this works, we can take this over my PR, no problem.

Reply:
(If this indeed is the cause of the problem,) ideally the OS image should be refreshed to contain a new version, so that we don’t have to do this at runtime. But I can only find https://cirrus-ci.org/guide/macOS/#list-of-available-images , nothing about the time when they were built or if they are rebuilt regularly.